clappr/clappr-ios
# Rename Jump plugin to QuickSeek

### Goal

To change JumpPlugin (and all others "Jump classes") to QuickSeekPlugin 

### How to test
- Run all Clappr tests
- Check if I forgot to rename something
- Run the application and use the QuickSeek Feature
